README.md

```diff
@@ -94,6 +94,7 @@ tkn pipeline start operator-hosted-pipeline \
   --param git_fork_url=https://github.com/MarcinGinszt/operator-pipelines-test.git \
   --param git_repo_url=https://github.com/redhat-openshift-ecosystem/operator-pipelines-test.git \
   --param git_username=test_user \
+  --param git_commit=0aeff5f71e4fc2d4990474780b56d9312554da5a \
```

Comment:
We should be able to get commit hash from already provided params.

Reply:
No need to, since it's provided by webhook. It would be an unnecessary effort to write and maintain this logic.
However, new parameter should be included in the webhook definition: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/ansible/roles/operator-pipeline/tasks/operator-hosted-pipeline-trigger.yml#L77

Reply:
This will be also needed in the release pipeline and calling a one git command is I guess easier than updating params in few places. I know it is available through webhooks, but as you can see this is an example situation when new param was created and forgotten to add in triggers.